code/BestBuyScraper.py

```diff
@@ -49,7 +49,7 @@ def check_stock_price(self, url):
         try:
             page = requests.get(url, headers=headers, timeout=5)
             # parsing the content of the page
-            soup = BeautifulSoup(page.text, "lxml")
+            soup = BeautifulSoup(page.text, "html.parser")
```

Comment:
What was the exact issue here?

Reply:
lxml has some other dependencies which were not installed. So I changed it to another parser that is used for AmazonScrapper and it also works here. I have manually tested this.